js/pages/submission/SubmissionStep1FormHandler.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @defgroup js_pages_submission
+ */
+// Create the namespace.
+jQuery.pkp.pages.submission = jQuery.pkp.pages.submission || { };
+
+/**
+ * @file js/pages/submission/SubmissionStep1FormHandler.js
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionStep1FormHandler
```

Comment:
We need this handler in `templates\controllers\modals\submissionMetadata\form\issueEntrySubmissionReviewForm.tpl` also as the section can be selected there too (review process -> show metadata dialog). How about moving this code to a base class and reusing it in submission and review process?

Reply:
What is the use-case for showing the section policy there? My understanding is that the Section Policy informs the submitting author about the requirements of submission to a section. 

Is there any reason to show this information after submission?

Reply:
I see now that you mean to display the Section Policy for the reviewer before they accept a review. Is that right? Can you give some examples of information in the policy that is important to a reviewer?

Reply:
> I see now that you mean to display the Section Policy for the reviewer before they accept a review. Is that right?

Yes, the reviewer is able to change the section in the metadata dialog before the submission is accepted in the review process. This means that the person that is able to change the section might not be aware of a policy or the policy details. I think it's a bit cumbersome for this person to manually go to the about submissions page to read the policy (or to instruct this person to got go there before changing the section). From my point of view consistency is important hence if the policy is displayed together with the section drop-down in the submission wizard then it should be that way in other forms also (where one can change the section to another value).

Reply:
Maybe we are talking about different things? If I login as a reviewer and open a review assignment, I am not able to change the section:

![selection_005](https://user-images.githubusercontent.com/2306629/47004820-461df480-d12a-11e8-87da-640ee8d0e411.png)

Is it different for you?

Reply:
Sorry, i'm not that familiar with the OJS wording yet, I was referring to "Journal editor". This user type can edit the meta data before the submission is send to review.

Reply:
Ah ok! So, my approach to UX is to remove as many distractions and unnecessary information for the most common interactions. Any information, even if small, will increase the cognitive load that is required each time we interact with this modal.

As an editor, I will look at this screen dozens or hundreds of times in a year. Of those times, I may only switch sections less than a tenth of the time. Furthermore, as an editor, if I need to switch a section, I am probably familiar with that section's policies. Maybe I need to refresh my memory a few times. But i'll still only need the section policies a tiny fraction of the time that I'm interacting this screen.

For that reason, I don't think that we should add the section policy into this metadata modal. If we felt that the section policy was something that would be needed more often, we could show a link to it. But I really don't feel that it's warranted in this case. If you disagree, I'm happy to take the question back to our UI/UX team to double-check my assumptions.

Reply:
Ok, fair enough.